tests/test_cpuinfo.py

```diff
@@ -8,40 +9,76 @@
 def test_cpuinfo(prog):
     cpuinfo.print_cpu_info(prog)
 
-    if not (ProgramFlags.IS_LIVE & prog.flags) or not (
-        prog["init_uts_ns"].name.machine.string_().decode("utf-8") == "x86_64"
-    ):
+    if not (ProgramFlags.IS_LIVE & prog.flags):
         return
```

Comment:
Let's go ahead and use `pytest.skip()` here. This is just a bit more honest about what's happening in the test - we can't test it so we're skipping the rest of it. Rather than returning, which indicates "we were able to test it, and it passed!"

By doing this after we've run `print_cpu_info()`, we still get the "smoke test" out of it -- if an error with the types, etc, crops up, the test would fail before it gets skipped.

```suggestion
        pytest.skip("Supported only on live systems")
```

You'll need an import at the top as well:
```python
import pytest
```

[Documentation](https://docs.pytest.org/en/7.0.x/how-to/skipping.html)

Reply:
Actually, this is fine as is. I'm going to merge it now :)